ivy/functional/frontends/jax/lax/operators.py

```diff
@@ -673,3 +673,20 @@ def cbrt(x):
 @to_ivy_arrays_and_back
 def tie_in(x, y):
     return y
+
+
+@with_supported_dtypes(
+    {
+        "2.5.1 and below": (
+            "float16",
+            "float32",
+            "float64",
+        )
+    },
+    "paddle",
```

Comment:
The framework should always be Jax in the Jax frontend functions. Incase, paddle backend only supports these particular data-types you should add these to the `erf` implementation in the paddle backend. Basically, the idea is that the frontend testing pipeline uses this decorator to compute the data-types supported by the native function ( in this case `jax.lax.erfc`), so that the ground truth can be calculated using the native function with the correct set of supported dtypes. Hope that makes sense :)